### ISSUE TITLE
can we skip empty rows when importing cognate sets?

### ISSUE THREAD

**Issue by nataliacp** [Anaphory/lexedata#224]
In the last arawak import, a completely empty row in the cognates.xlsx caused a completely empty row in cognatesets.csv. Can we skip completely empty rows in import? The empty row is still in the cognate googlesheet of arawak just after ARM25.

**Comment by Anaphory** [comment]
Oh, that empty row has upset me at times, too. Yes, we should skip those.

**Comment by Anaphory** [comment]
I have now spent a bit trying to find the best spot to filter the data so that an empty row will be ignored, but I don't know. @PaprikaSteiger do you have a good idea?

**Comment by Anaphory** [comment]
Oof, I think I found why it doesn't work yet.

https://github.com/Anaphory/lexedata/blob/77fd4f68332fdcca92dfa0a16377b8f2be9ad9dd/src/lexedata/importer/cognates.py#L31-L37

Line 36 is supposed to take care of this. Except. If there are fields that have a separator, line 33 kicks in. And if the entry for such a field in the cell is the empty string `''`, it gets split. Into `['']`. Which happens to not be an empty list, so suddenly there is some non-empty value in properties and the script thinks its a cognateset worth outputting :facepalm: 

So now I found this, now I can fix it.
